packages/react-ui/src/stories/common/Paper.stories.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { makeStyles, Paper, Typography } from '@material-ui/core';
+
+export default {
+  title: 'Common/Paper',
+  component: Paper,
+  argTypes: {
+    elevation: {
+      control: {
+        type: 'number',
+      }
+    }
+  }
+}
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: 'flex',
+    flexWrap: 'wrap',
+    '& > *': {
+      margin: theme.spacing(1),
+      width: theme.spacing(16),
+      height: theme.spacing(16),
+      backgroundColor: '#fafafa'
```

Comment:
We must keep colors from theme for Storybook stories (in this case don't change `Paper` bg color), and use themed colors when needed as well.